frontend-react/src/pages/home/Home.tsx

```diff
@@ -45,6 +46,29 @@ export const Home = () => {
                     );
                 })}
 
+                <div className="tablet:margin-bottom-8">
+                    <h2 className="font-sans-lg tablet:font-sans-xl margin-top-0 tablet:margin-bottom-0">
+                        How it works
+                    </h2>
+                    <p className="usa-intro margin-top-1 text-base">
+                        Our open-source platform aggregates and securely
+                        delivers health care data test results from
+                        organizations and testing facilities directly to public
+                        health entities.
+                    </p>
+                    <img
+                        src="/assets/img/ReportStreamDiagram_2022.png"
+                        alt="Organizations and testing facilities submit data, which ReportStream validates, augments and standardizes. Then, the data is routed as ReportStream determines the destination by subscription management. After this, it is transformed and transported to public health agencies in the desired data model and batch size."
+                    />
+                    <p>
+                        While ReportStream currently works with COVID-19 and
+                        mpox data, we are adding more reportable conditions
+                        soon. Contact us if you would like updates on additional
+                        conditions as they become available. Learn how to send
```

Comment:
Add a line break between "Contact us if you would like..." and "Learn how to send..." sentences